<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.batch.core.scope.StepScopeTests" tests="11" skipped="0" failures="0" errors="0" timestamp="2017-03-23T15:02:44" hostname="Michaels-MacBook-Pro-3.local" time="0.006">
  <properties/>
  <testcase name="testGetWithNothingAlreadyThere" classname="org.springframework.batch.core.scope.StepScopeTests" time="0.001"/>
  <testcase name="testOrder" classname="org.springframework.batch.core.scope.StepScopeTests" time="0.0"/>
  <testcase name="testRegisterDestructionCallback" classname="org.springframework.batch.core.scope.StepScopeTests" time="0.001"/>
  <testcase name="testName" classname="org.springframework.batch.core.scope.StepScopeTests" time="0.0"/>
  <testcase name="testGetConversationId" classname="org.springframework.batch.core.scope.StepScopeTests" time="0.0"/>
  <testcase name="testRegisterAnotherDestructionCallback" classname="org.springframework.batch.core.scope.StepScopeTests" time="0.001"/>
  <testcase name="testGetWithNoContext" classname="org.springframework.batch.core.scope.StepScopeTests" time="0.001"/>
  <testcase name="testParentContextWithSameStepExecution" classname="org.springframework.batch.core.scope.StepScopeTests" time="0.0"/>
  <testcase name="testGetWithSomethingAlreadyInParentContext" classname="org.springframework.batch.core.scope.StepScopeTests" time="0.001"/>
  <testcase name="testGetWithSomethingAlreadyThere" classname="org.springframework.batch.core.scope.StepScopeTests" time="0.001"/>
  <testcase name="testRemove" classname="org.springframework.batch.core.scope.StepScopeTests" time="0.0"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
